Wrappers/Python/ccpi/viewer/utils/conversion.py

```diff
@@ -968,397 +1175,87 @@ def RequestData(self, request, inInfo, outInfo):
 
         return 1
 
-    def GetOutput(self):
-        return self.GetOutputDataObject(0)
+class cilRawResampleReader(cilBaseResampleReader, cilBaseRawReader):
+    '''vtkAlgorithm to load and resample a raw file to an approximate memory footprint
+    '''
 
+    def __init__(self):
+        VTKPythonAlgorithmBase.__init__(self, nInputPorts=0, nOutputPorts=1)
+        cilBaseReader.__init__(self)
+        cilBaseResampleReader.__init__(self)
+        cilBaseRawReader.__init__(self)
```

Comment:
These 3 init lines should be replaced by 
`super(cilRawResampleReader, self).__init__()`